docs/realtime/realtimetransform.md

```diff
@@ -86,3 +88,8 @@ A common case where folks want this functionality is in order to pick up an obje
 Visually, this gives you the same result and will avoid bugs related to reparenting.
 
 *Tip: If the RealtimeTransform object has a Rigidbody on it, make sure to use the `MovePosition()` and `MoveRotation()` methods in `FixedUpdate()`*
+
+#### Skipping interpolation
+When teleporting a RealtimeTransform that has interpolation enabled (ex teleporting the player) it can be useful to suspend interpolation for that motion. Otherwise the transform will fly towards the destination instead of snapping instantly to it.
```

Comment:
```suggestion
When teleporting a RealtimeTransform that has interpolation enabled (ex. teleporting the player), it can be useful to suspend interpolation for that motion. Otherwise the transform will fly towards the destination instead of snapping instantly to it.
```